### DISCUSSION TITLE
Ability to Change Install Location

### DISCUSSION THREAD

**Discussion by picklegame** [stoatchat/discussions#140] [Feature Suggestions]
An option to change where the desktop app installs/stores data would be greatly appreciated as someone with a jam packed (C:) drive.

**Comment by JLUsr** [comment]
I tend to like installing applications and all related files within the same folder. Sometimes this has me installing portable versions of applications purely because the installable version of said application does not support designating where these files go. I won't necessarily say Stoat needs a portable for-desktop version (though it would be nice); but, I will say that having control over where files are stored is very important for many users.
